src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.cs

```diff
@@ -88,9 +101,16 @@ ProjectComponents GetSDKProjectComponents()
                     }
 
                     if (Sdk.Equals(MSBuildConstants.DesktopSdk, StringComparison.OrdinalIgnoreCase) ||
-                        GetPropertyValue("UseWPF").Equals("true", StringComparison.OrdinalIgnoreCase) ||
+                        GetPropertyValue("UseWPF").Equals("true", StringComparison.OrdinalIgnoreCase))
```

Comment:
I think with the SDK, WPF is only used if UseWPF is true and similarly for WinForms, so the Sdk.Equals check should be in its own if statement that sets ProjectComponents.WindowsDesktop but not WPF or WinForms.

Reply:
Oops, definitely shouldn't be an `or` statement here. Thanks Mike.